packages/suite/src/reducers/suite/suiteReducer.ts

```diff
@@ -35,11 +35,17 @@ export interface Flags {
     dashboardAssetsGridMode: boolean; // dashboard UI
 }
 
+export enum BitcoinUnits {
+    BTC,
+    Sats,
+}
+
 export interface SuiteSettings {
     theme: {
         variant: Exclude<SuiteThemeVariant, 'system'>;
     };
     language: Locale;
+    BitcoinUnits: BitcoinUnits;
```

Comment:
i would rather move it to reducers/wallet/settingsReducer  instead where we keep all settings related to wallet (like fiat currecy), suiteReducer is targeting general settings of the app

and for consitence use names starting with lowercase

Reply:
done